README.md

```diff
@@ -1,4 +1,262 @@
+[RFC issue tracker], and occasionally posting "pre-RFCs" on the
+[ipfs discussion forum](https://discuss.ipfs.io) for early review.
+
+As a rule of thumb, receiving encouraging feedback from long-standing project
+developers, and particularly members of the relevant [working group] is a good
+indication that the RFC is worth pursuing.
+
+
+## The RFC life-cycle
+[The RFC life-cycle]: #the-rfc-life-cycle
+
+In short, to get a major feature added to IPFS, one must first get the RFC
+merged into the RFC repository as a markdown file. At that point the RFC is
+"Accepted" and may be implemented with the goal of eventual inclusion into the corresponding protocols or libraries.
+
+As it grows from an idea to shipping in a supported release, an SDK RFC will traverse along:
+
+1. **Draft**: The owner of the SDK RFC has started to draft up how the subject will be handled and may be reviewing with a core group.  Comments are certainly welcome at this stage even though the owner hasn't worked through enough details to ask for...
+2. **Review**: This SDK RFC is in a review period. Stakeholders and the owner may still be iterating on some final details before signoff. A minimum review period has been defined.
+3. **Final Call**
+4. **Accepted**: All stakeholders have signed off and this SDK RFC is now or will be implemented soon.
+
+
+### Status: Draft
+
+When you've done the preparation described in [Before creating an RFC] and are ready to submit your RFC, create the pull request following these steps:
+
+
+  - Fork the RFC repo [RFC repository]
+  - Copy `0000-template-rfc.md` to `rfcs/0000-my-feature.md` (where "my-feature" is
+    descriptive. don't assign an RFC number yet).
+  - Fill in the RFC. Put care into the details: RFCs that do not present
+    convincing motivation, demonstrate understanding of the impact of the
+    design, or are disingenuous about the drawbacks or alternatives tend to be
+    poorly-received.
+  - Submit a pull request. As a pull request the RFC will receive design
+    feedback from the larger community, and the author should be prepared to
+    revise it in response.
+
+### Status: Review
+
+Once a an RFC has been submitted, it's ready for Review.
+
+  - Each pull request will be labeled with the most relevant [working group], which
+    will lead to its being triaged by that team in a future meeting and assigned
+    to a member of the working group.
+  - At this point, the status in the RFC markdown document should be marked "Review"
+  - Build consensus and integrate feedback. RFCs that have broad support are
+    much more likely to make progress than those that don't receive any
+    comments. Feel free to reach out to the RFC assignee in particular to get
+    help identifying stakeholders and obstacles.
+  - The working group will discuss the RFC pull request, as much as possible in the
+    comment thread of the pull request itself. Offline discussion will be
```

Comment:
I would change

> Offline discussion will be summarized on the pull request comment thread

To

> Offline discussion **must** be summarized on the pull request comment thread

Reply:
Just an idea: to remove ambiguity and make interpretation easier for non-native speakers we may consider prefacing the document with this disclaimer:

> The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and  "OPTIONAL" in this document are to be interpreted as described in  [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt)

Reply:
✅  done.